packages/account/src/Components/poa/status/unverified/unverified.tsx

```diff
@@ -1,22 +1,25 @@
-import React from 'react';
+import React, { ReactElement } from 'react';
 import { Button, Icon, Text } from '@deriv/components';
 import { localize } from '@deriv/translations';
 import IconMessageContent from '../../../icon-message-content';
 
 type TUnverified = {
+    title?: string | ReactElement;
+    description?: string | ReactElement;
```

Comment:
```suggestion
    description?: ReactNode;
```
if you want a more compact typedef, `ReactNode` already includes `string`